app/main/posts/posts-routes.js

```diff
@@ -86,22 +86,34 @@ function (
             }
         }
     )
+
+    /**
+     * @uirouter-refactor
+     * we need to  be able to set mode-context as collectionModeContext or as savedSearchModeContext
+     * without the route since the user would land in /views/map after the redirect
+     */
     .state(
         {
             url: '^/savedsearches/:savedSearchId/data',
             name: 'posts.data.savedsearch',
-            onEnter: ['PostFilters', 'savedSearch', function (PostFilters, savedSearch) {
+            onEnter: function ($state, $transition$, PostFilters, savedSearch) {
```

Comment:
You need to declare dependencies here ie `['PostFilters', 'savedSearch', function (PostFilters, savedSearch) {`
otherwise this breaks in production builds